docs/educationdata.ado

```diff
@@ -1110,6 +1111,7 @@ mata
 		token_cmd = tokens(summaries)
 		var_to_agg = token_cmd[2]
 		agg_by = ""
+		token_cmd = select(token_cmd, token_cmd[1,.]:!="year")
```

Comment:
Double checking: This line takes "year" out of the tokenized macro list if it's present?

Reply:
Yes, that's right.